<?xml version="1.0"?>
<launch>

  <node 
      pkg="vision_guided_planning" 
      type="spherical_capturing"
      name="spherical_capturing"  
      output="screen" >
  </node>
  
</launch>